src/cript/nodes/core.py

```diff
@@ -98,9 +122,12 @@ def _from_json(cls, json_dict: dict):
         # The call to the constructor might ignore fields that are usually not writable.
         try:
             node = cls(**arguments)
+        # TODO we should not catch all exceptions if we are handling them, and instead let it fail
```

Comment:
We are not catching the exception here.
We are reraising it.
See 129.
But before we are printing some extra info that can help debugging.

It's not ideal, but it isn't catching the exception.